workflow_scripts/install_linux_proto.sh

```diff
@@ -0,0 +1,15 @@
+export NUMCORES=`grep -c ^processor /proc/cpuinfo`
+
+git clone https://github.com/protocolbuffers/protobuf.git
+cd protobuf
+git checkout 3.11.x
+git submodule update --init --recursive
+./autogen.sh
```

Comment:
https://github.com/protocolbuffers/protobuf/blob/master/src/README.md#binary-compatibility-warning

 --disable-shared and enable PIC.